src/main/java/com/michelin/cio/hudson/plugins/copytoslave/CopyToSlaveBuildWrapper.java

```diff
@@ -125,16 +129,14 @@ else if(getDescriptor().isRelativeToHomeEnabled() && RELATIVE_TO_HOME.equals(rel
             String includes = env.expand(getIncludes());
             String excludes = env.expand(getExcludes());
 
-            listener.getLogger().printf("[copy-to-slave] Copying '%s', excluding %s, from '%s' on the master to '%s' on '%s'.\n",
+            listener.getLogger().printf("[copy-to-slave] Copying '%s', excluding %s, from '%s' on the master to '%s' on '%s'.%n",
                     includes, StringUtils.isBlank(excludes) ? "nothing" : '\'' + excludes + '\'', rootFilePathOnMaster.toURI(),
                     projectWorkspaceOnSlave.toURI(), Computer.currentComputer().getNode().getDisplayName());
 
-            // HUDSON-7999
-            MyFilePath.copyRecursiveTo(
-                    rootFilePathOnMaster,
-                    includes,
-                    excludes,
-                    isFlatten(), isIncludeAntExcludes(), projectWorkspaceOnSlave);
+            // JENKINS-7999 is fixed, so now use Jenkins FilePath instead of the
+            // custom MyFilePath included in this plugin
+            rootFilePathOnMaster.copyRecursiveTo(includes, excludes,
+                    projectWorkspaceOnSlave);
```

Comment:
This removes silently the `flatten` feature from https://issues.jenkins-ci.org/browse/JENKINS-8220